### DISCUSSION TITLE
saved chunks and future updates.

### DISCUSSION THREAD

**Discussion by jessefaler** [mircokroon/minecraft-world-downloader#142] [Q&A]
where do I find the saved chunks? what is the directory?

do you plan to update this to work for 1.17

**Comment by mircokroon** [comment]
They are saved to `world` in the same path as the downloader itself by default. You can also view the folder by clicking "Open" in the World tab, if you're using the GUI:
![image](https://user-images.githubusercontent.com/23699979/112461781-91f40f80-8d60-11eb-8ae5-b2997a442e35.png)

Yes, I'll update it to 1.17 once they are getting around to pre-releases. It's not really worth doing for each snapshot.

**Reply by vrogojin** [reply]
By some reason, I can see in MC single player the chunks saved earlier by MWD only within MWD render distance around the last saved player position. If player moves beyond that radius, the world is empty. Meanwhile, MWD still seem to be able to stream to client saved chunks beyond that render distance when travelling in multiplayer.  Is this bug or feature? Does it have anything to do with the fact that MWD got chunks from MC server 1.16.5 but not 1.16.2?

Many thanks!

**Reply by mircokroon** [reply]
Hmm I don't think that should be the case, this might happen if you are in spectator mode with the gamerule enable to prevent chunk generation, e.g. if you checked the `Prevent chunk generation` box.
